src/features/token-operations/models/token-operation.ts

```diff
@@ -1,10 +1,14 @@
 import { PaymentAgreement } from "@/features/payment-agreements/models";
 
 export class TokenOperation {
+  _id: string;
```

Comment:
Do not use the `_id` field. All schemas have a virtual `id` property that is returned with the API calls.
If we migrate to another database type, we won't have the `_id` property.